package.json

```diff
@@ -52,6 +53,7 @@
     "react-select": "^5.7.0",
     "remark-gfm": "^1.0.0",
     "sns-validator": "^0.3.4",
+    "styled-components": "^5.3.6",
```

Comment:
ran into this error when starting and it goes away after installing it.
```

error - ./node_modules/react-data-table-component/dist/index.cjs.js:1:68
Module not found: Can't resolve 'styled-components'

Import trace for requested module:
./src/common/components/table.tsx
./pages/facilitator/index.tsx

https://nextjs.org/docs/messages/module-not-found
error - ./node_modules/react-data-table-component/dist/index.cjs.js:1:68
Module not found: Can't resolve 'styled-components'
```

It could be because of one the dependencies ? but I am not sure ,  I can't seem to find anywhere where we require from styled compoenents directly

Reply:
./node_modules/react-data-table-component/dist/index.cjs.js:1:68
Module not found: Can't resolve 'styled-components'

Import trace for requested module:
./src/common/components/table.tsx
./pages/facilitator/index.tsx

https://nextjs.org/docs/messages/module-not-found

this is the error message, must be from the new table ?

Reply:
Oh ok, weird that they wouldn't just make it a dependency of their own. No worries then 👍